This switches from file-based auth to pam-based out for basic auth.
--- etc/openwsman.conf
+++ etc/openwsman.conf
@@ -4,7 +4,7 @@
 ssl_cert_file = /etc/openwsman/servercert.pem
 ssl_key_file = /etc/openwsman/serverkey.pem
 digest_password_file = /etc/openwsman/digest_auth.passwd
-basic_password_file = /etc/openwsman/simple_auth.passwd
+#basic_password_file = /etc/openwsman/simple_auth.passwd
 
 min_threads = 4
 max_threads = 10
@@ -15,8 +15,8 @@
 # Authentication backend for BASIC authentication. Default is to read a configuration file defined with 'basic_password_file'
 #
 
-#basic_authenticator = libwsman_pam_auth.so
-#basic_authenticator_arg = openwsman
+basic_authenticator = libwsman_pam_auth.so
+basic_authenticator_arg = openwsman
 
 
 [client]
